DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: bibtex
-Version: 0.4.3
+Version: 0.4.3.9000
```

Comment:
Bump, this can be undone

Reply:
Let's go 0.5.0 for the next CRAN release and, then, bump to 1.0.0.

Reply:
Done